geopyspark/geotrellis/rasterio.py

```diff
@@ -0,0 +1,91 @@
+import os
+import math
+import geopyspark as gps
+try:
+    import rasterio
+except ImportError:
+    raise ImportError("rasterio must be installed in order to use the features in the geopyspark.geotrellis.rasterio package")
+
+
+__all__ = ['read_windows', 'get']
+
+# On driver
+_GDAL_DATA = os.environ.get("GDAL_DATA")
+
+def crs_to_proj4(crs):
+    """Converts RasterIO CRS to proj4 string using osgeo library.
+
+    Args:
+        crs: :class:`rasterio.crs.CRS`
+
+    Returns:
+        Proj4 string of CRS.
+    """
+    try:
+        from osgeo import osr
+    except ImportError:
+        raise ImportError("osgeo must be installed in order to use the crs_to_proj4 function")
+
+    srs = osr.SpatialReference()
+    srs.ImportFromWkt(crs.wkt)
+    proj4 = srs.ExportToProj4()
+    return proj4
+
+def read_windows(uri, xcols=512, ycols=512, bands=[1], crs_to_proj4=crs_to_proj4):
+    """Given a URI, this method uses rasterio to generate series of windows of the desired dimensions.
+
+    Args:
```

Comment:
The `Args` section of the docs should be expanded so there similar to the other docs in the library. For example, https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark/geotrellis/geotiff.py#L32-L76